homeassistant/components/overkiz/translations/en.json

```diff
@@ -1,34 +1,34 @@
 {
     "config": {
-        "step": {
-            "user": {
-                "description": "The Overkiz platform is used by various vendors like Somfy (Connexoon / TaHoma), Hitachi (Hi Kumo), Rexel (Energeasy Connect) and Atlantic (Cozytouch). Enter your application credentials and select your vendor.",
-                "data": {
-                    "username": "Username",
-                    "password": "Password",
-                    "hub": "Hub"
-                }
-            }
+        "abort": {
+            "already_configured": "Account is already configured"
         },
         "error": {
             "cannot_connect": "Failed to connect",
-            "too_many_requests": "Too many requests, try again later.",
             "invalid_auth": "Invalid authentication",
             "server_in_maintenance": "Server is down for maintenance.",
+            "too_many_requests": "Too many requests, try again later.",
             "unknown": "Unexpected error"
         },
-        "abort": {
-            "already_configured": "Device is already configured"
+        "step": {
+            "user": {
+                "data": {
+                    "hub": "Hub",
+                    "password": "Password",
+                    "username": "Username"
+                },
+                "description": "The Overkiz platform is used by various vendors like Somfy (Connexoon / TaHoma), Hitachi (Hi Kumo), Rexel (Energeasy Connect) and Atlantic (Cozytouch). Enter your application credentials and select your vendor."
+            }
         }
     },
     "options": {
         "step": {
             "update_interval": {
-                "title": "Update Interval",
-                "description": "The Overkiz integration periodically retrieves new events. Change the update interval to a lower value if you want more frequent updates, for example when you also control your devices outside Home Assistant.",
                 "data": {
                     "update_interval": "Update interval (in seconds)"
-                }
+                },
+                "description": "The Overkiz integration periodically retrieves new events. Change the update interval to a lower value if you want more frequent updates, for example when you also control your devices outside Home Assistant.",
```

Comment:
I don't think that it makes sense. If it's local and the device supports it, just stick to 30s.

Reply:
Just to clarify, it’s a cloud API. But indeed Somfy agreed on this 30s interval. For integration with only stateless devices (RTS), like the official API we can set this interval to 1h.

Reply:
Yeah, in that case let's drop the interval configuration and do what Somfy wants. 

We don't want to allow people to spam Somfy (and be identified as using HA!).

Reply:
I will remove the update interval in a follow up PR to this feature branch, lets set it to 30 seconds (as agreed) and set this to 1 hour+ when the user only has stateless RTS devices.

By the way, in the current way people cannot spam Somfy since the minimum interval is set to the agreed interval (30 seconds). This way users can choose to not do external polling HTTP requests every 30 seconds, when they don't control their devices outside Home Assistant.

If people really miss this feature, we will hear soon enough and it is just a simple PR to add it back. I agree that we should not complicate it by default. 👍🏻

Reply:
Fix proposed in https://github.com/home-assistant/core/pull/49639.